requirements.txt

```diff
@@ -1,8 +1,8 @@
 databricks-sql-connector>=3.1.0, <3.2.0
 dbt-spark~=1.8.0
-dbt-core~=1.8.0
-dbt-adapters~=1.2.0
+dbt-core>=1.8.0, <2.0
+dbt-adapters>=1.3.0, <2.0
 databricks-sdk==0.17.0
 keyring>=23.13.0
 pandas<2.2.0
```

Comment:
This whole file could just be `.`

Reply:
Will investigate that as a separate PR, thanks for bringing to my attention